pkg/controllers/provisioning/scheduling/node.go

```diff
@@ -102,3 +158,22 @@ func (n Node) hasCompatibleResources(resourceList v1.ResourceList, it cloudprovi
 	}
 	return true
 }
+
+func (n *Node) recalculateDaemonResources() {
+	n.daemonResources = nil
+	for _, ds := range n.daemons {
```

Comment:
I was a bit thrown by ds, since I read it to be daemonset. I think it's actually daemon. 
```suggestion
	for _, daemon := range n.daemons { // or d
```